### PR TITLE
Refactor backend: Remove implicit classes

### PR DESCRIPTION
**Pull request by fm3**
### Summary
Implicit classes were a scala2 workaround to extend existing types. We can do that with scala3 extensions now (or go for explicit methods where it make sense).

### Steps to test:
- Upload an NML with some skeleton trees, should still work

------
- [x] Removed dev-only changes like prints and application.conf edits
- [x] Considered [common edge cases](../blob/master/.github/common_edge_cases.md)
- [x] Needs datastore update after deployment

### CODE REVIEW

File: app/models/annotation/nml/NmlParser.scala [hunk 1]
```diff
@@ -26,14 +26,13 @@ import com.scalableminds.webknossos.tracingstore.tracings.skeleton.{MultiCompone
 import com.typesafe.scalalogging.LazyLogging
 import models.annotation.{SharedParsingParameters, UploadedVolumeLayer}
 import models.dataset.DatasetDAOLike
-import com.scalableminds.util.tools.Box._
-import com.scalableminds.util.tools.{Box, Empty, Failure, Full}
+import com.scalableminds.util.tools.Box.*
```
Comment (MichaelBuessemeyer) [inline app/models/annotation/nml/NmlParser.scala]: Side note: Maybe not use * import here to narrow down whats actually needed for faster compilation?

**Review by MichaelBuessemeyer — APPROVED** [review]
Neat, looking good :tada:
